imhamt/src/lib.rs

```diff
@@ -420,17 +363,35 @@ mod tests {
         (h, reference)
     }
 
-    #[quickcheck]
-    fn plan_equivalent(xs: Plan<String, u32>) -> bool {
-        let (h, reference) = arbitrary_hamt_and_btree(xs, next_u32, |v| v.wrapping_mul(2));
-        property_btreemap_eq(&reference, &h)
+    fn arbitrary_hamt_and_btree(
+    ) -> impl Strategy<Value = (Hamt<DefaultHasher, String, u32>, BTreeMap<String, u32>)> {
+        any_with::<Vec<PlanOperation>>(size_range(1000..5120).lift())
```

Comment:
I don't see why the size range must start at 1000 here. It should be possible to create a Hamt with 0 operations? On failures, we want reduction to the smallest failing input.

Reply:
Makes sense 👍